loadtest/azuredeploy.parameters.loadtest.defaults.json

```diff
@@ -11,6 +11,7 @@
         "fileServerDiskCount":         { "value": 2 },
         "fileServerDiskSize":          { "value": 128 },
         "redisDeploySwitch":           { "value": false },
-        "sshPublicKey":                { "value": "__SSH_PUB_KEY__" }
+        "sshPublicKey":                { "value": "__SSH_PUB_KEY__" },
+        "searchType":	               { "value": "azure-search" }
```

Comment:
Please remove this. We don't want azure-search on our load testing (at least yet).